pkg/kube/secretcontroller/secretcontroller.go

```diff
@@ -39,8 +39,13 @@ const (
 	maxRetries = 5
 )
 
-// Required for unit test override.
-var loadKubeConfig = clientcmd.Load
+// LoadKubeConfig is a unit test override variable for loading the k8s config.
+// DO NOT USE - TEST ONLY.
+var LoadKubeConfig = clientcmd.Load
+
+// CreateInterfaceFromClusterConfig is a unit test override variable for interface create.
+// DO NOT USE - TEST ONLY.
+var CreateInterfaceFromClusterConfig = kube.CreateInterfaceFromClusterConfig
```

Comment:
since this is a passed-in parameter, I don't see need to add this variable

Reply:
Similar to above this is not mocking the passed in variable, but for mocking line 237 of the file.